heudiconv/info.py

```diff
@@ -24,6 +24,7 @@
 
 EXTRA_REQUIRES = {
     'tests': TESTS_REQUIRES,
+    'extras': ['dcmstack'],
```

Comment:
sadly not on pypi

how about adding a `dependency_links` option to `setuptools.setup` with the gitURL?

Reply:
eh, yesterday that I have seen travis to fail I remembered that patched dcmstack is needed.  we have it patched in neurodebian IIRC but indeed, we cannot just add it from pypi :-/  I will try to reach @moloney  again, hope that he is alright (previous attempts went without a reply)
I was not aware of `dependency_links`, is it present in olderish setuptools as well?

Reply:
meanwhile pushed a commit removing it from requires to get tests passing.  We could decide how to proceed on that one later (situation wouldn't be worse than before)